### DISCUSSION TITLE
Filenames with special characters?

### DISCUSSION THREAD

**Discussion by a-schneider-aries** [ImageMagick/ImageMagick#7338] [Help]
### ImageMagick version

7.1.1-Q16-HDRI

### Operating system, version and so on

Azure VM, windows 

### Description

I'm working through a bug where images with special character (specifically '%') in the filename are not building correctly. I know that certain special characters are not supported in text with imagemagick. Does this extend to filenames as well?

**Comment by dlemstra** [comment]
File names with `%` should be supported. Are you working with batch files? Do you have more details on what kind of issue you have?

**Comment by snibgo** [comment]
These commands work fine for me, in Windows CMD:
```
magick rose: x%.png

magick x%.png x2.png
```

@a-schneider-aries: please show the exact command that gives problems, and the shell used (bash, or Windows BAT, or Windows CMD, etc).

If % is a special character in your shell (eg Windows BAT), you will need to escape it.
